dji_robomaster_ep_driver/include/dji_robomaster_ep_driver/joystick_controller_mappings.hpp

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#if DJI_ROBOMASTER_EP_DRIVER__SUPPORTED_ROS_VERSION == 2
+#include <geometry_msgs/msg/twist.hpp>
+#include <sensor_msgs/msg/joy.hpp>
+#elif DJI_ROBOMASTER_EP_DRIVER__SUPPORTED_ROS_VERSION == 1
+#include <geometry_msgs/Twist.h>
+#include <sensor_msgs/Joy.h>
+#else
+#error "Undefined or unknown DJI_ROBOMASTER_EP_DRIVER__SUPPORTED_ROS_VERSION"
+#endif
+
+namespace dji_robomaster_ep_driver
+{
+class ControllerMapping
+{
+public:
+#if DJI_ROBOMASTER_EP_DRIVER__SUPPORTED_ROS_VERSION == 2
+  using Twist = geometry_msgs::msg::Twist;
+  using Joy = sensor_msgs::msg::Joy;
+#elif DJI_ROBOMASTER_EP_DRIVER__SUPPORTED_ROS_VERSION == 1
+  using Twist = geometry_msgs::Twist;
+  using Joy = sensor_msgs::Joy;
+#endif
+
+  ControllerMapping(double max_linear_velocity, double max_angular_velocity)
+    : max_linear_velocity_(max_linear_velocity),
+      max_angular_velocity_(max_angular_velocity)
```

Comment:
nit This can be simplified to
```
    max_angular_velocity_(max_angular_velocity) {}
```

Reply:
Done in https://github.com/ToyotaResearchInstitute/tri_hardware_drivers/pull/9/commits/da68bcad3b4e1fd867b00f81f68282515dfaf19e.